donate/templates/pages/core/landing_page.html

```diff
@@ -1,3 +1,50 @@
 {% extends "pages/core/landing_page_master.html" %}
+{% load static wagtailcore_tags wagtailimages_tags i18n %}
 
-{# There are no overrides in this template, but alternative apps can template-overide this file (e.g. thunderbird) #}
+{% block content %}
+
+<div class="layout layout__fundraise-up">
+    <div class="layout__container">
+        <div class="layout__half-col">
+            <div class="image-feature column-spacing">
+                {% block featured_image %}
+                    <div class="image-feature__container">
+                        <picture class="image-feature__image">
+                        {% image page.featured_image fill-290x95 as imageMobile %}
+                        {% image page.featured_image fill-580x190 as imageMobile_2x %}
+                        {% image page.featured_image width-350 as imageTablet %}
+                        {% image page.featured_image width-700 as imageTablet_2x %}
+                        {% image page.featured_image width-690 as imageDesktop %}
+                        {% image page.featured_image width-1380 as imageDesktop_2x %}
+                        <source media="(min-width: 1201px)" srcset="{{ imageDesktop.url }}, {{ imageDesktop_2x.url }} 2x" />
+                        <source media="(min-width: 600px)" srcset="{{ imageTablet.url }}, {{ imageTablet_2x.url }} 2x" />
+                        <source srcset="{{ imageMobile.url }}, {{ imageMobile_2x.url }} 2x" />
+
+                        <img src="{{ imageMobile.url }}" class="image-feature__image"/>
```

Comment:
Missing indentation :)

```suggestion
                            {% image page.featured_image fill-290x95 as imageMobile %}
                            {% image page.featured_image fill-580x190 as imageMobile_2x %}
                            {% image page.featured_image width-350 as imageTablet %}
                            {% image page.featured_image width-700 as imageTablet_2x %}
                            {% image page.featured_image width-690 as imageDesktop %}
                           {% image page.featured_image width-1380 as imageDesktop_2x %}    
                            <source media="(min-width: 1201px)" srcset="{{ imageDesktop.url }}, {{ imageDesktop_2x.url }} 2x" />
                            <source media="(min-width: 600px)" srcset="{{ imageTablet.url }}, {{ imageTablet_2x.url }} 2x" />
                            <source srcset="{{ imageMobile.url }}, {{ imageMobile_2x.url }} 2x" />

                           <img src="{{ imageMobile.url }}" class="image-feature__image"/>
```

Reply:
Thanks Mavis! Added 🙌